homeassistant/components/mysensors/cover.py

```diff
@@ -14,6 +15,15 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class CoverState(Enum):
+    """An enumeration of the standard cover states."""
+
+    OPEN = 0
+    OPENING = 1
+    CLOSED = 2
+    CLOSING = 3
```

Comment:
Let's make the sequence go from open to closed.
```suggestion
    CLOSING = 2
    CLOSED = 3
```

Reply:
Makes sense to me. You never know if someone will want to use the ordinals for something.